frontend/src/pages/ExperimentList.test.tsx

```diff
@@ -28,6 +28,10 @@ import { ReactWrapper, ShallowWrapper, shallow } from 'enzyme';
 import { RoutePage, QUERY_PARAMS } from '../components/Router';
 import { range } from 'lodash';
 import { ButtonKeys } from '../lib/Buttons';
+import { NamespaceContext } from 'src/lib/KubeflowClient';
+import { render } from '@testing-library/react';
+
```

Comment:
nit: worth having a comment above to explain the field name?

Reply:
Yes, I see why you are confusing.
Will do

Reply:
Added